packages/components/src/test/properties/NumericPropertyInput.test.tsx

```diff
@@ -0,0 +1,268 @@
+import { EmptyLocalization } from "@itwin/core-common";
+import { IModelApp } from "@itwin/core-frontend";
+import { Presentation } from "@itwin/presentation-frontend";
+import sinon from "sinon";
+import { NumericInput, NumericPropertyInput, NumericPropertyInputAttributes } from "../../presentation-components/properties/NumericPropertyInput";
+import { render, waitFor } from "@testing-library/react";
+import { createRecord } from "./NumericPropertyEditor.test";
+import { createRef } from "react";
+import { PrimitiveValue } from "@itwin/appui-abstract";
+import { expect } from "chai";
+import userEvent from "@testing-library/user-event";
+
+describe("<NumericPropertyInput />", () => {
+  beforeEach(async () => {
+    const localization = new EmptyLocalization();
+    sinon.stub(IModelApp, "initialized").get(() => true);
+    sinon.stub(IModelApp, "localization").get(() => localization);
+    await Presentation.initialize();
+  });
+
+  afterEach(async () => {
+    Presentation.terminate();
+    sinon.restore();
+  });
+
+  it("get value from target input reference", async () => {
```

Comment:
```suggestion
  it("renders with initial value", async () => {
```

Reply:
I think that we should change to `NumericPropertyInput`